packages/code-studio/src/styleguide/MockIrisGridTreeModel.ts

```diff
@@ -235,6 +242,62 @@ class MockIrisGridTreeModel
   isValidForCell(column: ModelIndex, row: ModelIndex, value: string): boolean {
     return true;
   }
+
+  async columnStatistics(column: Column): Promise<ColumnStatistics> {
+    throw new Error('Not defined in mock');
+  }
+
+  async commitPending(): Promise<void> {
+    throw new Error('Not defined in mock');
+  }
+
+  async export(): Promise<Table> {
+    throw new Error('Not defined in mock');
+  }
+
+  get pendingDataErrors(): Map<number, Error> {
+    throw new Error('Not defined in mock');
+  }
+
+  get rollupConfig(): RollupConfig {
+    throw new Error('Not defined in mock');
+  }
+
+  set rollupConfig(rollupConfig: RollupConfig) {
+    throw new Error('Not defined in mock');
+  }
+
+  get selectDistinctColumns(): string[] {
```

Comment:
Can't the return here be `never[]` since it always throws?

Reply:
Yep. I changed all the methods that throw in the mock to return `never` or `Promise<never>` as needed.